Week06/lesson12/homework/rock_paper_scissors_starter/js/main.js

```diff
@@ -0,0 +1,81 @@
+var humanChoice = "";
+var humanScore = 0;
+var computerScore = 0;
+
+
+$('.choice').click(function() {
+   humanChoice = this.id;
+
+	 var choices = ["rock", "paper", "scissors"];
+
+	 var computerChoice = choices[Math.floor(Math.random() * choices.length)];
```

Comment:
Excellent job working out how to randomise the choices! And storing them in an array to boot!

Might be nice to extract this into a function though, to tidy up a little:

```
function getComputerChoice() {
  var choices = ["rock", "paper", "scissors"];
  return choices[Math.floor(Math.random() * choices.length)];
}

$(".choice").click(function() {
  var humanChoice = this.id;
  var computerChoice = getComputerChoice();
  // etc..
}
```